.gitignore

```diff
@@ -40,6 +40,7 @@ build/Release
 # Dependency directories
 node_modules/
 jspm_packages/
+package-lock.json

```

Comment:
I thought package-lock.json files were supposed to be committed in a repository?

Reply:
I was testing something and left it there. Removed!